src/github.js

```diff
@@ -2,22 +2,35 @@ import GitHubApi from 'github'
 import { getUserSettings } from './utils'
 
 export const loadUserRepos = (callback) => {
-    const userSettings = getUserSettings()
-    if (!userSettings) return
+    const userSettings = getUserSettings();
```

Comment:
Could you remove the extra `;` that got added, just to keep consistency with the rest of the code (which I don't think has them right now)?